### ISSUE TITLE
Link to docs page is broken

### ISSUE THREAD

**Issue by colin-mccarthy** [ansible/ansible-lint#402]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible --version
ansible-lint --version
```

- ansible installation method: one of source, pip, OS package
- ansible-lint installation method: one of source, pip, OS package

# Desired Behaviour

ansible-lint/docs/docsite/README.md

Line 3:

`Contains the source behind [docs.ansible.com/lint](https://docs.ansible.com/lint)`

Please give some details of the feature being requested
or what should happen if providing a bug report

Please fix the link to the docs page, I would have submitted a PR fixing it but was not able to find where it existed online.

Possible security bugs should be reported via email to `security@ansible.com`

# Actual Behaviour (Bug report only)

Please give some details of what is actually happening.
Include a [minimum complete verifiable example](http://stackoverflow.com/help/mcve)
with:
- playbook
- output of running ansible-lint
- if you're getting a stack trace, output of
  `ansible-playbook --syntax-check playbook`
